src-web/nls/platform.properties

```diff
@@ -177,8 +177,9 @@ ansible.scheduleMode.disabled.title = Disable automation
 ansible.scheduleMode.disabled.info = Automation does not run until this setting is updated.
 ansible.unsaved.data = You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.
 ansible.loading.info = Loading policy violation automation data...
-ansible.create.success = New policy violation automation ${0} is successfully created on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
-ansible.edit.success = Existing policy violation automation ${0} is successfully updated on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
+ansible.manual.success = The automation has been manually triggered successfully. Please see history tabe to check ansible history information.
```

Comment:
typo tab`e`

Reply:
@dockerymick can you review the strings here?

Reply:
@ycao thanks for pulling me in. 

I suggest the following updates:

* Can we use only one period in line 179?
* The automation has been manually initiated successfully. View the History tab to check the ansible history information.